packages/block-library/src/verse/index.js

```diff
@@ -15,7 +15,7 @@ export const name = 'core/verse';
 export const settings = {
 	title: __( 'Verse' ),
 
-	description: __( 'A block for haiku? Why not? Blocks for all the things! (See what we did here?)' ),
+	description: __( 'Insert haiku. Use special spacing formats. Or quote song lyrics. (See what we did here?)' ),
```

Comment:
This is not a valid haiku, as far as I can tell. The first line has 4 syllables, but the third line has 5 syllables. They should both have the same number of syllables.